### PR TITLE
Update glossary.json

### PR DESCRIPTION
**Pull request by adrianbrink**

### CODE REVIEW

File: json/glossary.json [hunk 1]
```diff
@@ -2,25 +2,25 @@
   {
     "id": "abci",
     "title": "ABCI",
-    "body": "ABCI is the Application BlockChain Interface...",
+    "body": "ABCI is the Application BlockChain Interface that provides a clean interface between a state transition machine on one computer and the mechanics of its replication across multiple computers.",
```
Comment (jolesbi) [inline json/glossary.json]: can you use a different word other than interface again?

File: json/glossary.json [hunk 2]
```diff
@@ -32,37 +32,37 @@
   {
     "id": "cosmos-hub",
     "title": "Cosmos Hub",
-    "body": "The Cosmos Hub is...",
+    "body": "The Cosmos Hub is the multi-asset ledger that enforces invariants and maintain the security between connected zones.",
```
Comment (jolesbi) [inline json/glossary.json]: “maintains”

File: json/glossary.json [hunk 3]
```diff
@@ -32,37 +32,37 @@
...
   {
     "id": "cosmos-ui",
     "title": "Cosmos UI",
-    "body": "The Cosmos UI is...",
+    "body": "The Cosmos UI is the graphical interface for end users to interract with the Cosmos Network",
     "url": ""
   },
   {
     "id": "delegator",
     "title": "Delegator",
```
Comment (jolesbi) [inline json/glossary.json]: we started using the word “delegates” (noun) to describe people who can be delegated too (including validators and candidates). can you add this to the glossary?

File: json/glossary.json [hunk 4]
```diff
@@ -74,7 +74,7 @@
   {
     "id": "ethermint",
     "title": "Ethermint",
-    "body": "Ethermint is...",
+    "body": "Ethermint is the Ethereum Virtual Machine plugged on top of the Tendermint Consensus Engine. It is natively compatible with Cosmos",
```
Comment (jolesbi) [inline json/glossary.json]: is “plugged” the strongest word here?

File: json/glossary.json [hunk 5]
```diff
@@ -92,13 +92,13 @@
...
     "title": "Gaia",
-    "body": "Gaia is the...",
+    "body": "Gaia is the preliminary version of the Cosmos Hub. It is based on the Cosmos-SDK.",
     "url": ""
   },
   {
     "id": "governance",
     "title": "Governance",
-    "body": "Blockchain governance is...",
+    "body": "Blockchain governance is a process that allows participant to vote on-chain using their staked tokens",
```
Comment (jolesbi) [inline json/glossary.json]: “participants”

File: json/glossary.json [hunk 6]
```diff
@@ -92,13 +92,13 @@
   {
     "id": "gaia",
     "title": "Gaia",
-    "body": "Gaia is the...",
+    "body": "Gaia is the preliminary version of the Cosmos Hub. It is based on the Cosmos-SDK.",
```
Comment (zramsay) [inline json/glossary.json]: Maybe: "It uses the building blocks of the Cosmos SDK?" Drop the `-` from `Cosmos-SDK` for consistency.

File: json/glossary.json [hunk 7]
```diff
@@ -110,7 +110,7 @@
   {
     "id": "ibc",
     "title": "IBC",
-    "body": "IBC is the interblockchain communication protocol...",
+    "body": "IBC is the Inter- Blockchain Communication Protocol. It allows fast-finality blockchains to exchange value with eachother",
```
Comment (zramsay) [inline json/glossary.json]: `Inter- Block` dash typo
`eachother` => `each other`

File: json/glossary.json (hunk 1 above)
Comment (zramsay) [reply json/glossary.json]: communication layer?

File: json/glossary.json [hunk 8]
```diff
@@ -2,25 +2,25 @@
...
     "title": "BFT",
-    "body": "BFT is Byzantine Fault-Tolerance...",
+    "body": "BFT or Byzantine Fault-Tolerance is the ability to tolerate machines failing in arbitrary ways, including becoming malicious.",
     "url": ""
   },
   {
     "id": "blockchain",
     "title": "Blockchain",
-    "body": "Blockchain are...",
+    "body": "A blockchain, originally block chain, is a continuously growing list of records, called blocks, which are linked and secured using cryptography.",
```
Comment (zramsay) [inline json/glossary.json]: what's meant by "originally block chain", like, it used to be spelled as two words but now it's one? can probably drop that part

File: json/glossary.json [hunk 9]
```diff
@@ -32,37 +32,37 @@
...
     "title": "Cosmos UI",
-    "body": "The Cosmos UI is...",
+    "body": "The Cosmos UI is the graphical interface for end users to interract with the Cosmos Network",
     "url": ""
   },
   {
     "id": "delegator",
     "title": "Delegator",
-    "body": "Delegators are...",
+    "body": "Delegators are Atom holders that contribute to the security of the network by choosing secure and highly available validators through the process of delegation.",
```
Comment (zramsay) [inline json/glossary.json]: shouln'dt we have a field describing candidates?

File: json/glossary.json (hunk 4 above)
Comment (zramsay) [reply json/glossary.json]: "...is the EVM running on top of Tendermint Consensus, allowing for significantly reduced block times and scalability. It is natively ..."
- maybe also define block time in the glossary?

**Comment by nylira** [comment]
@zramsay @jolesbi Can you guys edit the PR yourself (or create a PR for e.g. candidates) for the changes you are requesting? Let's consider aib-data sort of like Wikipedia, where anyone on the team can edit as they see fit.

**Comment by zramsay** [comment]
done. @jolesbi can you approve/merge?

**Comment by jolesbi** [comment]
i can approve but i can't merge

**Comment by nylira** [comment]
@jolesbi I tried adding you as a collaborator. Try and see if it works next time.

**Pull request merged**